### ISSUE TITLE
iPad version: Timer page

### ISSUE THREAD

**Issue by dshr** [toggl/mobileapp#4504]
Implement max width of the time entry table and running TE indicator, along with correct positioning of the Start TE button

**Comment by sofokli1** [comment]
I think we should wait for the grouped time entries to be done and merged in develop, and get develop into the ipad branch before changing cell constraints. We will end up in useless conflicts. What do you think @dshr ?

**Comment by dshr** [comment]
Just saw this comment 80% through #4505 😅 While we might get conflicts, so far there have been no substantial changes to `.xib` files in the Grouped TEs project, so we might be in the clear. Seeing how both of us have already done some work, I say we keep these for a bit and 🦑 them only after the Grouped TE project is done and merged into `develop`.
